{
  "comments": [
    {
      "key": {
        "uuid": "6eb64e82_9c6138f7",
        "filename": "include/sdbusplus/asio/property.hpp",
        "patchSetId": 5
      },
      "lineNbr": 24,
      "author": {
        "id": 1000900
      },
      "writtenOn": "2021-12-15T22:24:18Z",
      "side": 1,
      "message": "So we cannot pass an existing std::function by lvalue reference. Probably an inconsequential restriction.",
      "range": {
        "startLine": 24,
        "startChar": 65,
        "endLine": 24,
        "endChar": 67
      },
      "revId": "68f7db95da145322f519b8712de3c897a8a5aceb",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "3bd822cc_d860464e",
        "filename": "include/sdbusplus/asio/property.hpp",
        "patchSetId": 5
      },
      "lineNbr": 29,
      "author": {
        "id": 1000900
      },
      "writtenOn": "2021-12-15T22:24:18Z",
      "side": 1,
      "message": "nit: any reason we don\u0027t start using std::error_code? Mixing the two will result in some extra copies, but otherwise they seem seamlessly convertible.",
      "range": {
        "startLine": 29,
        "startChar": 39,
        "endLine": 29,
        "endChar": 64
      },
      "revId": "68f7db95da145322f519b8712de3c897a8a5aceb",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}